run.py

```diff
@@ -99,12 +99,14 @@ def get_iss_cmd(base_cmd, elf, log):
   return cmd
 
 
-def gen(test_list, simulator, simulator_yaml, output_dir, sim_only,
-        compile_only, lsf_cmd, seed, cwd, cmp_opts, sim_opts, timeout_s):
+def gen(test_list, csr_file, isa, simulator, simulator_yaml, output_dir, sim_only,
+        compile_only, lsf_cmd, seed, cwd, cmp_opts, sim_opts, iterations, timeout_s):
   """Run the instruction generator
 
   Args:
     test_list      : List of assembly programs to be compiled
+    csr_file       : YAML file containing description of all CSRs
+    isa            : Processor supported ISA subset
```

Comment:
It misses document for `iterations`, as well as `parser.add_argument` for this option?

Reply:
The add_argument() call exists, but you're right, the documentation is not. Thanks for pointing that out, will add that in.